src/components/events/components/TagsDisplay.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+/** @jsxImportSource @emotion/react */
+import "twin.macro";
```

Comment:
Is there a way to import this at a higher level so we don't need to call it everytime we want to use tw?

Reply:
no there isn't not from anything ik atm